sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -134,6 +136,17 @@ def get_physical_port_name_dict(logical_port_name, port_mapping):
 
     return port_name_dict
 
+# Get dictionary of asic to set of corresponding logical ports
+def get_asic_to_logical_ports_dict(port_mapping):
```

Comment:
should we move this function to src/sonic-platform-daemons/sonic-xcvrd/xcvrd/xcvrd_utilities/port_mapping.py and make it a member of PortMapping class?

Reply:
I did not put it in PortMapping class since this class is more of tied to a single port and the corresponding mapping for it such as its physical/logical port number or asic ID.
However, the function get_asic_to_logical_ports_dict is tied to a ASIC and it maps to a set of ports. Hence, I added this in xcvrd and not as a member of PortMapping class.